### PR TITLE
Docs for IG headers propagation

### PR DESCRIPTION
**Pull request by rachitchauhan43**
Links to https://github.com/kserve/kserve/pull/2396

## Proposed Changes 

- Adds `Features` section to existing IG doc 
- Explains how to configure headers propagation

### CODE REVIEW

**Comment by rachitchauhan43** [comment]
Since, the feature [PR](https://github.com/kserve/kserve/pull/2396) has been merged. Can we mere this documentation PR too ?

File: docs/modelserving/inference_graph/README.md [hunk 1]
```diff
@@ -35,3 +35,26 @@ leverage these strengths to build `InferenceGraph` and enable users to deploy co
...
+
+#### Headers Propagation
+If you want IG router to propagate the headers, you passed in the request to IG, to all the steps in your graph then you can do so using `inferenceservice-config`
+config-map in kserve namespace.
+For example:
+If you want to propagate a certain header, say "Custom-Header", then you can edit the `router` section of `inferenceservice-config`
+config-map like this :
+```
+{
+    "image" : "kserve/router:v0.9.0",
```
Comment (yuzisun) [inline docs/modelserving/inference_graph/README.md:48]:
```suggestion
    "image" : "kserve/router:v0.10.0",
```

File: docs/modelserving/inference_graph/README.md [hunk 2]
```diff
@@ -35,3 +35,26 @@ leverage these strengths to build `InferenceGraph` and enable users to deploy co
...
+    "memoryRequest": "100Mi",
+    "memoryLimit": "1Gi",
+    "cpuRequest": "100m",
+    "cpuLimit": "1"
```
Comment (yuzisun) [inline docs/modelserving/inference_graph/README.md:52]:
```suggestion
    "cpuLimit": "1",
```

File: README.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# KServe documentation
+#2 KServe documentation
```
[the remark this answers is not in the document]
Comment (rachitchauhan43) [reply README.md]: Yep. Fixed it. I guess, i was testing something regarding my github profiles setting to have proper sign off and this sneaked in.

**Pull request closed without merging**